src/Discord.Net.Commands/Attributes/RemarksAttribute.cs

```diff
@@ -3,6 +3,9 @@
 namespace Discord.Commands
 {
     // Extension of the Cosmetic Summary, for Groups, Commands, and Parameters
+    /// <summary>
+    ///     An attribute that allows you to attach remarks to your command.
```

Comment:
I would suggest rephrasing the summary for both the attributes. A summary should begin with a verb like all the other summaries in this lib.

References:
* https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=netframework-4.7.2
* https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=netframework-4.7.2